### PR TITLE
added google analytics

### PR DESCRIPTION
**Pull request by sidthe**
before going with further debates on the needs of the crowbar website in terms of content and effort, we need to understand where is it standing right now. Specifically, where do users come from, referred by which sites and how many.

### CODE REVIEW

**Comment by aplanas** [comment]
Good idea. In openSUSE we are using PIWIK (http://beans.opensuse.org/piwik). Others projects like Machinery are using PIWIK too.

-1 for Google Analytics

File: index.html [hunk 1]
```diff
@@ -29,6 +29,23 @@
...
+                (i[r].q = i[r].q || []).push(arguments)
+            }, i[r].l = 1 * new Date();
+            a = s.createElement(o),
+                m = s.getElementsByTagName(o)[0];
+            a.async = 1;
+            a.src = g;
+            m.parentNode.insertBefore(a, m)
+        })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
+
+        ga('create', 'UA-75091706-1', 'auto');
```
Comment (toabctl) [inline index.html:45]: Who's owning this tracker-ID? Do we have a company (or Cloud) account for doing such things?

**Comment by tboerger** [comment]
:-1: for Google analytics, :+1: for PIWIK!

**Comment by sidthe** [comment]
Piwik hosted ain't free, is it? no one owns the tracker-id. I just have a separate google account created for this purposes.

**Comment by aplanas** [comment]
@sidthe As I pointed out in the other comment, there is a PIWIK installation deployed for openSUSE, and Machinery is using it too. PIWIK is a PHP and GPL license software.

**Comment by sidthe** [comment]
@aplanas aaa, ok :) I only realized. Perfect. Even better. So how do we get an account there?

**Comment by aplanas** [comment]
@sidthe I will talk with Darix. Or he have the admin account, or the can drive the process to create an account for Crowbar. He is part of the infra team.

Once the account is created, is a matter of C&P a javascript code.

**Comment by sidthe** [comment]
@aplanas yes,I used Piwik before inhouse in Nokia. I just did not realize we have a hosted one. Will you take it with Darix or shall I get in touch with him?

**Comment by aplanas** [comment]
@sidthe Please, do it. Darix is a SUSE employee (mruecker). You can find him on the floor service, and / or in the SUSE IRC channel. If he can't do it himself, he can point you to the real admin.

**Comment by toabctl** [comment]
@sidthe or just fill a ticket for the infra team

**Comment by sidthe** [comment]
it's fine, just dropped him a mail. I know the guy.

**Pull request closed without merging**